src/lib/errorHandler.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This code snippet handles any un-handled errors from occuring in the consumer iframe.
+ * This is a global event handler that will capture all the error messages and log them to the console.
+ * Once the error is logged to the console this onerror event handler will suppress the script error from surfacing on the UI,
```

Comment:
> script error from surfacing on the UI

I'd like to know more about this. What were you seeing surface?

Reply:
So the script error that you would see would be like the following : 
![MPagesScriptIssue_Error](https://user-images.githubusercontent.com/45810410/123869908-399c5300-d8f7-11eb-9440-67fddd4968b4.png)
I dont have screen shots for examples where Date() is undefined etc(since they are really hard to recreate), but for the most part when the JS is minified you would see the ```Object expected``` error with not much detail on it as shown above.

Reply:
So if the iFrames are constantly reloading when components are moved around it will result in  a barrage of such error popups one after another until the reloading is complete.